tests/testthat/test-bayesianpredictionperformance.R

```diff
@@ -0,0 +1,425 @@
+                                                                                                                                                                                                                                                                                                                                                     11.5, 9.7, 10, 15.03, 11.8, 11.4, 11.2, 13.3, 10.3, 11.2),
+                            pts.before.2010 = c(FALSE, TRUE, FALSE, TRUE, FALSE, TRUE,
+                                                TRUE, TRUE, TRUE, TRUE, FALSE, TRUE, TRUE, TRUE, TRUE, FALSE,
+                                                TRUE, TRUE, TRUE, FALSE, TRUE, FALSE, TRUE)), row.names = c(NA,
+                                                                                                            -23L), class = "data.frame")
+  results <- runAnalysis("BayesianPredictionPerformance", dataset, options)
+
+
+  test_that("Autocorrelations plot matches", {
+    plotName <- results[["results"]][["acPlot"]][["data"]]
+    testPlot <- results[["state"]][["figures"]][[plotName]][["obj"]]
+    jaspTools::expect_equal_plots(testPlot, "autocorrelations-plot-1")
+  })
+
+  test_that("Forest plot matches", {
+    plotName <- results[["results"]][["forestPlot"]][["data"]]
+    testPlot <- results[["state"]][["figures"]][[plotName]][["obj"]]
+    jaspTools::expect_equal_plots(testPlot, "forest-plot-1")
+  })
+
+  test_that("Autocorrelations plot matches", {
+    plotName <- results[["results"]][["gRPlot"]][["data"]]
+    testPlot <- results[["state"]][["figures"]][[plotName]][["obj"]]
+    jaspTools::expect_equal_plots(testPlot, "autocorrelations-plot-1")
+  })
+
+  test_that("Prior and posterior plot matches", {
+    plotName <- results[["results"]][["priorAndPosteriorPlot"]][["data"]]
+    testPlot <- results[["state"]][["figures"]][[plotName]][["obj"]]
+    jaspTools::expect_equal_plots(testPlot, "prior-and-posterior-plot-1")
+  })
+
+  test_that("Running means plot matches", {
+    plotName <- results[["results"]][["rmPlot"]][["data"]]
+    testPlot <- results[["state"]][["figures"]][[plotName]][["obj"]]
+    jaspTools::expect_equal_plots(testPlot, "running-means-plot-1")
+  })
+
+  test_that("Observed-Expected Ratio Meta-Analysis Summary table results match", {
+    table <- results[["results"]][["summaryTable"]][["data"]]
+    jaspTools::expect_equal_tables(table,
+                                   list(1.09088086726799, 0.88575857156341, 0.285932954156389, 1.34586691302466,
+                                        2.45439237587575))
+  })
+}
+
+# O:E: Poisson/log (+ asymmetry tests)
+{
```

Comment:
This does nothing, so please remove it everywhere.

Reply:
changed to wrapping by ----